DESCRIPTION

```diff
@@ -8,24 +8,28 @@ Authors@R: c(person("Steph", "Locke"
              , email = "troyhunt@hotmail.com"
              , role = c("aut")
              , comment="HaveIBeenPwned.com"
-             ))
+             ),
+             person("Locke Data", role = c("fnd")),
+             person("Maëlle", "Salmon", role = c("ctb")))
 Description: Utilising the 'Have I been pwned?' API (see <https://haveibeenpwned.com/API/v2>
     for more information), check whether email addresses and/or user names have been present
     in a publicly disclosed data breach.
 Depends:
     R (>= 3.2.3)
 License: MIT + file LICENSE
 LazyData: true
-Suggests:
+Suggests: 
     testthat,
     knitr,
-    rmarkdown
+    rmarkdown,
+    lintr
 Imports:
     httr,
     urltools,
     jsonlite,
     ratelimitr
-RoxygenNote: 6.0.1
+RoxygenNote: 6.0.1.9000
 VignetteBuilder: knitr
-URL: https://github.com/stephlocke/HIBPwned

```

Comment:
Maybe better to just refer to the pkgdown site

Reply:
But we have the github in the bugs field, but meh, I don't have particularly strong feelings about this so sure thing, we'll go with both